tests/test-create-invalid-fedora29.cmd

```diff
@@ -0,0 +1 @@
+footloose create --config %testName.static
```

Comment:
Just a note: while it's ok to have one integration test to make sure the validation is bubbling up to the user, validation is one of those things that should be done with unit tests (not now, it's ok to add them when we have more than one check).

Reply:
I take note